### ISSUE TITLE
Tooltip behind the new <dialog /> element

### ISSUE THREAD

**Issue by marekvanicek** [Akryum/floating-vue#999]
I am using v-tooltip inside the new html-native <dialog /> element. But the tooltip is hidden behind the backdrop. Is there anything that can be done about this? Mutating z-index does not really work.

<img width="1144" alt="Snímek obrazovky 2023-11-14 v 11 58 12" src="https://github.com/Akryum/floating-vue/assets/63812970/c0e2d2b5-3510-4a76-88e2-9b993f76e888">

**Comment by linpan** [comment]
me, too.

**Comment by meirroth** [comment]
same

**Comment by BrunoGGM** [comment]
+1

**Comment by marekvanicek** [comment]
@BrunoGGM dont think this has a fix. Dialogs completely ignore z-indexes. Went back to custom-made modal solution.

**Comment by leonardospeca** [comment]
It seems solved here -> https://github.com/floating-ui/floating-ui/issues/1842
Maybe could be implemented also in this library?

**Comment by MiniDigger** [comment]
stumbled upon this, the issue is that floating-vue teleports the tooltip to the bottom of the body, I assume to prevent stacking context issues. however dialogs create a new stacking context in their top layer, so this breaks.
luckily floating-vue allows you to customize this teleporting behavior. sadly its not fully documented, but you can look at the code here https://github.com/Akryum/floating-vue/blob/19857764c4f73dea7ed44a7d970adb968ee7ad90/packages/floating-vue/src/components/Popper.ts#L873-L875

tl;dr: set the container prop to false.
